test/contact-summary.spec.js

```diff
@@ -37,8 +39,9 @@ describe('getContactSummary', () => {
     const contact = {};
     const reports = [];
     const lineage = [];
-    await harness.getContactSummary(contact, reports, lineage);
-    expect(functionStub.args[0]).to.deep.eq([contact, reports, lineage]);
+    const cht = coreAdapter.chtScriptApi(['default_roles']);
+    await harness.getContactSummary(contact, reports, lineage, cht);
```

Comment:
the `harness.getContactSummary` interface is unchanged. this is not the correct fix for this test